brave/brave-core
# Fix Command+A command on Mac

If we pass 0 as keycode of NSEvent, it could be translated as
'a' key. So we should pass the actual keycode.


Resolves https://github.com/brave/brave-browser/issues/31059



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Open up multiple tabs
* Focus omnibox and type some text. Make sure you don't select all text already.
* Press Cmd + A or activate app menu > edit > select all
* Should select all texts.
